src/modules/storage/LedgerStorage.ts

```diff
@@ -39,7 +49,42 @@ export class LedgerStorage extends Storages
             signature TEXT,
             tx_count INTEGER NOT NULL,
             enrollment_count INTEGER NOT NULL
-        )`;
+        );
+
+        CREATE TABLE IF NOT EXISTS "transactions"
+        (
+            "block_height"      INTEGER NOT NULL,
+            "tx_index"          INTEGER NOT NULL,
+            "tx_hash"           TEXT NOT NULL,
+            "type"              INTEGER NOT NULL,
+            "inputs_count"      INTEGER NOT NULL,
+            "outputs_count"     INTEGER NOT NULL,
+            PRIMARY KEY("block_height", "tx_index")
+        );
+
+        CREATE TABLE IF NOT EXISTS "tx_inputs"
+        (
+            "block_height"      INTEGER NOT NULL,
+            "tx_index"          INTEGER NOT NULL,
+            "in_index"          INTEGER NOT NULL,
+            "previous"          TEXT NOT NULL,
+            "out_index"         INTEGER NOT NULL,
+            PRIMARY KEY("block_height", "tx_index", "in_index")
+        );
+
+        CREATE TABLE IF NOT EXISTS "tx_outputs"
+        (
+            "block_height"  INTEGER NOT NULL,
+            "tx_index"      INTEGER NOT NULL,
+            "output_index"  INTEGER NOT NULL,
+            "tx_hash"       TEXT NOT NULL,
+            "utxo_key"      TEXT NOT NULL,
+            "amount"        NUMERIC NOT NULL,
+            "address"       TEXT(56) NOT NULL,
+            "used"          TEXT(1) NOT NULL DEFAULT 'N',
```

Comment:
Why is this `TEXT` instead of just a bool or an integer? Also SQLite ignores these limits, it's not like in MySQL where it's a hard limit.

Reply:
Sorry there is no bool, but it should be an integer.

Reply:
Thanks. I will fix

Reply:
I changed, Thanks
``` SQL
            "address"       TEXT NOT NULL,
            "used"          INTEGER NOT NULL DEFAULT 0,
```